daos-stack/daos
# DAOS-3769 test: Added posix_stat.py to verify metadata of a file created in a container

Requirement: SRS-10-0303
DFS stat call reports meaningful information (i.e. valid file size, correct
block size, ...).

Test Steps:
Create files of different size (block size) in a mounted POSIX container and
verify the size with stat command. Also verify the creation time with date
command.

Test-tag: stat_parameters
Skip-func-hw-test: true
Skip-unit-test: true
Skip-func-test-leap15: true
Signed-off-by: Makito Kano <makito.kano@intel.com>
